include/deal.II/numerics/vector_tools_constraints.h

```diff
@@ -314,7 +324,9 @@ namespace VectorTools
 #else
          .ReferenceCell::get_default_linear_mapping<dim, spacedim>()
 #endif
-         ));
+         ),
+    const IndexSet &   refinement_edge_indices = IndexSet(),
+    const unsigned int level                   = numbers::invalid_unsigned_int);
```

Comment:
Further down in this file, we have a function to compute the tangential flux (actually two interfaces, one for the zero-constrained case which we call `compute_normal_flux_constraints`). It would be nice to have a symmetrical interface, i.e., have similar capability there.

Reply:
Same here, I will do it in a separate PR.